llvm/lib/Support/CMakeLists.txt

```
@@ -113,6 +113,7 @@ if(LLVM_INTEGRATED_CRT_ALLOC)
     add_compile_definitions(ENABLE_OVERRIDE ENABLE_PRELOAD)
     set(ALLOCATOR_FILES "${LLVM_INTEGRATED_CRT_ALLOC}/rpmalloc/rpmalloc.c")
     set(delayload_flags "${delayload_flags} ${WL}-INCLUDE:malloc")
+    string(STRIP ${delayload_flags} delayload_flags)
```

```suggestion
    set(delayload_flags "${delayload_flags}" "${WL}-INCLUDE:malloc")
```
or alternatively `list(APPEND delayload_flags "${WL}-INCLUDE:malloc")`. (I'd prefer the latter in new code, but the former is consistent with the existing code in this region).